src/index.test.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import {
+  render,
+  screen,
+} from '@testing-library/react';
+
+import { stripes } from '../package';
+
+import '../test/jest/__mock__';
+
+import CheckOutRouting from './index';
+
+const { route } = stripes;
+const testIds = {
+  checkOutComponent: 'checkout-component',
```

Comment:
Could you please change from `'checkout-component',` to `'checkoutComponent', `?